.coafile

```diff
@@ -39,3 +39,7 @@ language = python3
 # 0.8.x (current stable release) needs them.
 ci_keywords, keywords = \#TODO, \# TODO, \#FIXME, \# FIXME
 cs_keywords =
+
+[yml]
+bears = YAMLLintBear
+files = *.yml
```

Comment:
newline here.

Reply:
@Monal5031 I added newline at the end of .coafile :)
i understand, i'll never open new PRs in future, is there anything else how this PR can be improved?

Reply:
Yeah the changes look good, but I just noticed that we aren't runnig coala on travis. Should we make that change in this PR itself @Makman2 ?

Reply:
No that's okay, not task of this issue :+1: